packages/legend-application-studio/src/components/workspace-setup/WorkspaceSetup.tsx

```diff
@@ -57,13 +60,91 @@ import {
 import { debounce, guaranteeNonNullable } from '@finos/legend-shared';
 import { WorkspaceSetupStore } from '../../stores/workspace-setup/WorkspaceSetupStore.js';
 import { DocumentationLink } from '@finos/legend-lego/application';
+import { openShowcaseManager } from '../../stores/ShowcaseManagerState.js';
 
 const WorkspaceSetupStoreContext = createContext<
   WorkspaceSetupStore | undefined
 >(undefined);
 
 export const DEFAULT_WORKSPACE_SOURCE = 'HEAD';
 
+export const ShowcaseCard: React.FC = () => {
+  const applicationStore = useLegendStudioApplicationStore();
+  return (
+    <BaseCard
+      className="workspace-setup__content__card"
+      cardMedia={undefined}
+      cardName="Showcase projects"
+      cardContent="Review showcase projects with sample project code and re-use existing code snippets to quickly build your model."
+      cardActions={[
+        {
+          title: 'Showcase explorer',
+          content: <FileImportIcon />,
+          action: () => openShowcaseManager(applicationStore),
+        } as MuiCardActionConfig,
+      ]}
+      isStable={true}
+    />
+  );
+};
+
+export const SandboxCard: React.FC = () => {
+  const applicationStore = useLegendStudioApplicationStore();
+  const sandboxDocument = applicationStore.documentationService.getDocEntry(
+    LEGEND_STUDIO_DOCUMENTATION_KEY.APPLICATION_SANDBOX,
+  );
+  return (
+    sandboxDocument && (
+      <BaseCard
+        className="workspace-setup__content__card"
+        cardMedia={undefined}
+        cardName={sandboxDocument.title!}
+        cardContent={sandboxDocument.markdownText!.value}
+        cardActions={[
+          {
+            title: sandboxDocument.text!,
+            content: <FileImportIcon />,
+            action: () =>
+              applicationStore.navigationService.navigator.visitAddress(
+                sandboxDocument.url!,
+              ),
+          } as MuiCardActionConfig,
+        ]}
+        isStable={true}
+      />
+    )
+  );
+};
+
+export const RuleEngagementCard: React.FC = () => {
+  const applicationStore = useLegendStudioApplicationStore();
+  const ruleEngagementDocument =
+    applicationStore.documentationService.getDocEntry(
+      LEGEND_STUDIO_DOCUMENTATION_KEY.APPLICATION_RULE_ENGAGEMENT,
+    );
+  return (
+    ruleEngagementDocument && (
+      <BaseCard
+        className="workspace-setup__content__card"
+        cardMedia={undefined}
+        cardName={ruleEngagementDocument.title!}
```

Comment:
same comment as above with the `.title!`

Reply:
I update the condition to check if they are undefined